### PR TITLE
Troubleshoot Ceph network connection without nc

### PR DESCRIPTION
**Pull request by fultonj**
The `nc` command is not installed on the Glance pod so we cannot suggest it be used to test the pod's connection to Ceph. Instead the procedure is re-written using a Python socket since that has confirmed to be available in the Glance pod.

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -149,17 +149,45 @@ the [networking document](networking.md). The IPs and ports of the Ceph
 cluster's monitors should be in `/etc/ceph.conf`. From within the pod, read
 this and try to reach the IP as well as connect to the port.
 ```
-$ oc debug --container glance-api pod/glance-default-external-api-0
-bash-5.1$ cat /etc/ceph/ceph.conf
-# minimal ceph.conf for 7fbaf2fd-f70e-5625-b00e-65b33ee823a6
+$ oc get pods | grep glance | grep external-api-0
```
Comment (fmount) [inline troubleshooting.md:152]: I think we can remove `| grep glance` as we're going to get the same `Pod` in the example

**Review by fmount — COMMENTED** [review]
small nit, looks good otherwise!

File: troubleshooting.md (hunk 1 above)
Comment (fultonj) [reply troubleshooting.md]: done

**Review by fmount — APPROVED** [review]
/lgtm

**Pull request merged**